modules/ztp-sno-node-reboot-scenarios.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.adoc
+
+:_content-type: CONCEPT
+[id="ztp-sno-node-reboot-scenarios_{context}"]
+= SNO node reboot scenario
+
+On a {sno-caps} cluster and in {product-title} clusters generally a situation might arise due to a node reboot or kubelet restart where an associated device plugin restart has the affect of causing the application pod to fail with the `UnexpectedAdmissionError` error. This error is reported due to the fact that application pods that need devices come up before the pod that is serving the devices because there is no way to control the pod restart order.
```

Comment:
We have identified failure at kubelet restarts as a regression and is being worked on right now so let's slip mentioning that.
```suggestion
On a {sno-caps} cluster and in {product-title} clusters generally a situation might arise in case a node reboot occurs without node drain where an application pod requesting devices fails with the `UnexpectedAdmissionError` error. This error is reported due to the fact that application pods that need devices can come up before the pod that is serving the devices because there is no way to control the pod restart order.
```